src/java/test/org/apache/zookeeper/server/quorum/Zab1_0Test.java

```diff
@@ -37,6 +37,8 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.Map;
```

Comment:
Good catch, @afine. This test is related to ZOOKEEPER-1558. Perhaps, we need to re-look the fix to see any chance of creating snapshot with uncommitted state.